panther_hardware_interfaces/src/panther_system.cpp

```diff
@@ -0,0 +1,505 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <panther_hardware_interfaces/panther_system.hpp>
+
+#include <array>
+#include <chrono>
+#include <functional>
+#include <limits>
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <rclcpp/logging.hpp>
+
+#include <hardware_interface/types/hardware_interface_type_values.hpp>
+
+#include <panther_hardware_interfaces/utils.hpp>
+
+// todo: add user variable to the script that will trigger DOUT4 instead of safety stop
```

Comment:
Is it a task or a step in ClickUp? It just looks like something that will stay here forever :p

Reply:
There is no task for that and you're probably right xd We have to discuss it on the next daily.